components/brave_new_tab_ui/stories/default.tsx

```diff
@@ -3,22 +3,52 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
+import { Provider as ReduxProvider } from 'react-redux'
 import { storiesOf } from '@storybook/react'
-import { withKnobs, select, boolean } from '@storybook/addon-knobs/react'
+import { withKnobs, select } from '@storybook/addon-knobs/react'
+import Theme from 'brave-ui/theme/brave-default'
+import DarkTheme from 'brave-ui/theme/brave-dark'
+import BraveCoreThemeProvider from '../../common/BraveCoreThemeProvider'
 
 // Components
-import NewTabPage from './default/index'
+import NewTabPage from '../containers/newTab'
+import * as actions from '../actions/new_tab_actions'
+import store from '../store'
+import { getNewTabData } from './default/data/storybookState'
 
-storiesOf('New Tab/Default', module)
+export default function Provider ({ story }: any) {
+  return (
+    <ReduxProvider store={store}>
+      <BraveCoreThemeProvider
```

Comment:
This addition makes the `Theme` tab in Storybook useless for NTP. You can still toggle themes in the `knobs` tab. I'm not sure whether or not this is the right move. cc @petemill

Reply:
I would be tempted to leave the storybook provided theme provider so that we can use the theme tab consistently. Was there a reason to add it here?

Reply:
It seems that for the container to work I needed to declare it here explicitly along with redux provider. I couldn't make it work in both places in a timely manner. Maybe the theme tab is best for components rather than a container but I personally don't see difference or have a preference here. Choice made by convenience only

Reply:
ok sure